packages/quick_actions/quick_actions_ios/ios/Classes/QuickActionsPlugin.swift

```diff
@@ -80,12 +72,14 @@ public final class QuickActionsPlugin: NSObject, FlutterPlugin {
   public func applicationDidBecomeActive(_ application: UIApplication) {
     if let shortcutType = launchingShortcutType {
       handleShortcut(shortcutType)
-      launchingShortcutType = nil
+      self.launchingShortcutType = nil
     }
   }
 
-  private func handleShortcut(_ shortcut: String) {
-    channel.invokeMethod("launch", arguments: shortcut)
+  func handleShortcut(_ shortcut: String) {
+    self.testStub?()
```

Comment:
Is this to avoid relying on the Pigeon serialization details in the mock binary messenger?

If so, maybe we could instead do that for now since the TODO I requested above would address that in a better way long term, and we wouldn't need this test hack.

Reply:
Oh IC `IosQuickActionsFlutterApi` is a class rather than protocol. 

A workaround can be to wrap it with your own types: 

```
protocol IosQuickActionsFlutterApiWrapper {
  func launchAction(action: ...)
}

final class DefaultIosQuickActionsFlutterApiWrapper: IosQuickActionsFlutterApiWrapper {
  private let api: IosQuickActionsFlutterApi
  func launchAction(action: ...) {
    api.launchAction(action: action)
  }
}
```

Drawback is that this trivial wrapper itself is not unit testable in swift, but it should already be covered in dart tests.